components/brave_news/browser/resources/internals.tsx

```diff
@@ -21,14 +21,24 @@ const Grid = styled.div`
   gap: 8px;
 `
 
+const Blur = styled.div`
+  --edges: -128px;
+
+  z-index: -1;
+  position: fixed;
+  top: var(--edges); left: var(--edges); bottom: var(--edges); right: var(--edges);
```

Comment:
Q. Can `inset` be better?

```suggestion
  inset: var(--edges)
```

Reply:
Wow, I hadn't heard about that! That's so handy! New as of this year!